packages/contracts/deploy/L1/999-TransferOwnerships.ts

```diff
@@ -8,15 +8,17 @@ import { getDeploymentAddress } from '../../src/deploy-utils'
 
 const deployFn: DeployFunction = async (hre) => {
   const timelockProxyAddress = await getDeploymentAddress(hre, 'TimeLockProxy')
-
   await transferProxyAdminOwnership(hre, timelockProxyAddress)
-  await transferSystemConfigOwnership(hre, timelockProxyAddress)
+
+  const securityCouncilOwnerAddress = hre.deployConfig.securityCouncilOwner
+  await transferSecurityCouncilTokenOwnership(hre, securityCouncilOwnerAddress)
```

Comment:
```suggestion
  const securityCouncilTokenOwnerAddress =
    hre.deployConfig.securityCouncilTokenOwner
  await transferSecurityCouncilTokenOwnership(
    hre,
    securityCouncilTokenOwnerAddress
  )
```

Reply:
It is within 80 characters, so no newlines are required.

Reply:
It's not about new lines, about using securityCouncilTokenOwner. Please check the diff again.

Reply:
Oh I see.